tailcfg/tailcfg.go

```diff
@@ -1157,6 +1157,10 @@ type MapResponse struct {
 	// sees.
 	Health []string `json:",omitempty"`
 
+	// SSHPolicy, if non-nil, updates the SSH policy for how incoming
```

Comment:
update `CurrentMapRequestVersion`?

Reply:
Not yet. Once it's wired up to anything useful I can.